test_textbelt.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Not sure if we really need this. I tested directly in the terminal. Could we have this use the free daily API key `textbelt` instead of an actual API key?

Reply:
Agreed. I didn't mean to commit this file. We don't need it.